spec/models/user_spec.rb

```diff
@@ -3,6 +3,23 @@
 require "rails_helper"
 
 RSpec.describe User, type: :model do
+  describe "password" do
+    context "password 8 characters long" do
```

Comment:
`context` descriptions should start with `with/out` or `when` to make it more readable.
`it` descriptions should contain a verb.

Something like is easier to read:
```ruby
context "when the password is at least 8 chars long" do
  it "passes the validations" do
    # ...
  end
end
```

http://www.betterspecs.org/#contexts

Reply:
First: I agree that this leads to an improvement when reading the whole concatenated string of describe -> context -> it.

Having said that, I'm not sure that we should use betterspecs as a source of truth when it comes to questions regarding specs. Some of the things it encourages are a bit controversial to me (and not only me I think). For instance:
- Using subject to "dry tests up"
- Using `let`
- Using shared examples

I've seen all of those spiraling completely out of control, and it makes me think if we should have our own appendix to those guidelines, or if there's another guide that we can reference instead - or maybe I'm completely alone about this?

What are your thoughts? @heatherm @prrrnd @hcarreras @GeoffAbtion @runephilosof-abtion

Reply:
DRY shouldn't apply to specs in my junior opinion.
Making test be able to stand on their own and be readable is the way to go 👍

Reply:
I have strong opinions on many of these topics.
Remind me to post something about them when I'm not sick 🤒